geotrek/trekking/parsers.py

```diff
@@ -912,6 +917,30 @@ def _get_geom_from_gpx(data):
             geos.transform(settings.SRID)
             return geos
 
+    @staticmethod
+    def _get_file_with_good_encoding(file_name):
```

Comment:
AFAIU the returned value will always be the same filename as given in argument. So we could remove the `return` statement and rename this method as "_maybe_fix_encoding" so it is clear that the change is in-place.